README.md

```diff
@@ -1 +1,425 @@
+    responseWithMultiplePayloads = [{ StatusCode: 202, Payload: '' }, { StatusCode: 202, Payload: '' }]
+*/
+
+```
+
+#### CLIENT-CALL
+
+* `clientCall(functionName, clientCode, payload)` (*async*) : Invoke a function with a payload body and client. If multiple clients codes and/or payloads are send make one invoke for payload and client.
+    * `functionName` (*string*) **required**, function name in TitleCase or dash-case
+    * `clientCode` (*string* or *array of strings*) **required**, client code
+    * `payload` (*object* or *array of objects*), the data to send
+    * returns *array of objects*, with `StatusCode` and `Payload` fields (for each)
+
+```js
+'use strict'
+
+const { Invoker } = require('@janiscommerce/lambda');
+
+const responseOneClient = await Invoke.clientCall('AwakeKitties', 'katHouse');
+
+/*
+    Invoke JanisKittyService-readme-AwakeKitties function for 'katHouse' client (1 time)
+    responseOneClient = [{ StatusCode: 202, Payload: '' }]
+*/
+
+const responseTwoClient = await Invoke.clientCall('AwakeKitties', ['katHouse', 'katIsland']);
+
+/*
+    Invoke JanisKittyService-readme-AwakeKitties function 2 times, one for 'katHouse' client, other for 'katIsland'
+    responseTwoClient = [{ StatusCode: 202, Payload: '' }, { StatusCode: 202, Payload: '' }]
+*/
+
+const responseOneClientOnePayload = await Invoke.clientCall('AwakeKitties', 'katHouse', { name 'Tom' });
+
+/*
+    Invoke JanisKittyService-readme-AwakeKitties function for 'katHouse' client and 'Tom' payload (1 time)
+    responseOneClientOnePayload = [{ StatusCode: 202, Payload: '' }]
+*/
+
+const responseOneClientTwoPayload = await Invoke.clientCall('AwakeKitties', 'katHouse', [{ name: 'Tom' }, { name: 'Blacky' }]);
+
+/*
+    Invoke JanisKittyService-readme-AwakeKitties function 2 times for 'katHouse' client and 'Tom' payload and other for 'Blacky'
+    responseOneClientTwoPayload = [{ StatusCode: 202, Payload: '' }, { StatusCode: 202, Payload: '' }]
+*/
+
+const responseTwoClientOnePayload = await Invoke.clientCall('AwakeKitties', ['katHouse', 'katIsland'], { name 'Tom' });
+
+/*
+    Invoke JanisKittyService-readme-AwakeKitties function 2 times, one for 'katHouse' client and 'Tom' and other one for 'katIsland' client and 'Tom'
+    responseTwoClientOnePayload = [{ StatusCode: 202, Payload: '' }]
+*/
+
+const responseTwoClientTwoPayload = await Invoke.clientCall('CallKitties', ['katHouse', 'katIsland'], [{ name: 'Tom' }, { name: 'Blacky' }]);
+
+/*
+    Invoke JanisKittyService-readmeEAwakeKitties function  4 times, one for 'katHouse' and 'Tom', other for 'katHouse' and 'Blacky', other for 'katIsland' and 'Tom', other for 'katIsland' and 'Blacky'
+    responseTwoClientTwoPayload = [{ StatusCode: 202, Payload: '' }, { StatusCode: 202, Payload: '' }, { StatusCode: 202, Payload: '' }, { StatusCode: 202, Payload: '' }]
+*/
+
+```
+
+#### RECALL
+
+* `recall()` (*async*) : Invoke the function where it is called with the same
```

Comment:
Capaz así se entiende un poco más: Invokes the same function recursively
Ese with the same me hace ruido